test/remote/gateways/remote_braintree_token_nonce_test.rb

```diff
@@ -24,12 +24,20 @@ def setup
     }
   end
 
-  def test_client_token_generation
+  def test_authorization_fingerprint_generation
     generator = TokenNonce.new(@braintree_backend)
-    token = generator.client_token
+    token = generator.authorization_fingerprint
     assert_not_nil token
   end
 
+  def test_client_token_generation_with_mid
```

Comment:
❓ Question:

so Nraintree Sandbox doesn't care about a valid mid ?, you can just send a '1234' and they are going to provide you with a client token any way ?

Reply:
Correct, I tried passing multiple values ​​and got a successful result with every field sent including the known field